src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit';
+import books from '../../books';
+
+const initialState = {
+  books,
```

Comment:
- [ ] Nice work so far 👍🏿  creating an initial book state but you book state is currently not holding a data type please attribute the book to an empty array.

 `book:[]`